packages/editor/src/browser/editor-manager.ts

```diff
@@ -116,7 +116,10 @@ export class EditorManagerImpl implements EditorManager, WidgetFactory {
             const newEditor = new EditorWidget(textEditor, this.selectionService);
             newEditor.id = this.id + ":" + uri.toString();
             newEditor.title.closable = true;
+            /* label is what appears in the tab itself.  */
```

Comment:
are such comments useful? one should be able to look up APIs, no need to document them on the callee side

Reply:
I don't know, I like when there are comments that explain the intent of the code, so that when something does wrong, you know what the author was trying to do.  Even if it's possible to go look up an API, it's definitely not practical when reading the code, and it doesn't tell you what we are trying to achieve here.  Also, the documentation doesn't really help me understand where these strings are going to end up:

http://phosphorjs.github.io/phosphor/api/widgets/classes/title.html#label
http://phosphorjs.github.io/phosphor/api/widgets/classes/title.html#caption